Dockerfile

```diff
@@ -7,7 +7,11 @@ RUN go build -o bin/hiveutil github.com/openshift/hive/contrib/cmd/hiveutil
 RUN go build -o bin/hiveadmission github.com/openshift/hive/cmd/hiveadmission
 RUN go build -o bin/hive-operator github.com/openshift/hive/cmd/operator
 
-FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base as hive
+FROM registry.access.redhat.com/ubi7/ubi
```

Comment:
`registry.svc.ci.openshift.org/ocp/4.2:base` is built from that same ubi7 image.

Reply:
It cannot be pulled without auth, I think that's true for all ocp images.

Reply:
Ah. I remember you saying that before. Sorry for being so hard-headed.

Reply:
No problem!